CONTRIBUTING.md

```diff
@@ -47,30 +49,37 @@ This allows other members of the Nipype development team to confirm that you are
 
 **2. [Fork][link_fork] the [Nipype repository][link_nipype] to your profile.**
 
-This is now your own unique copy of Nipype.
+This is now your own unique copy of Nipype repository.
 Changes here won't affect anyone else's work, so it's a safe space to explore edits to the code!
 
-Make sure to [keep your fork up to date][link_updateupstreamwiki] with the master repository.
+You can clone your Nipype repository in order to create a local copy of the code on your machine.
+In your local Nipype directory, run `pip install -e .[dev]`.
+This will add your version of nipype to your local python environment and
+install dependencies needed for development.
+
+Make sure to [keep your fork up to date][link_updateupstreamwiki] with the original Nipype repository.
```

Comment:
Maybe we can introduce/reinforce the "upstream" vocabulary by replacing `original` with `"upstream"`.

Reply:
I tried to not used work "upstream" on purpose, just to minimize number of new words (you don't need to have "upstream" to keep your master up to date). However the link is about "upstream", so I would just add an extra link that describes it:
"Make sure to keep your fork up to date with the original Nipype repository, one way of doing it is to [configure a new remote upstream repository](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [syncing your fork with the upstream][link_updateupstreamwiki].

Reply:
That seems fine. Again, though, the comma is doing a bit too much work. I would make it a new sentence at "One way".

Also, you're switching tenses in this sentence with "doing", "to configure" and "syncing". As a suggestion (with some other slight rewording): "One way to do this is to configure a new remote named "upstream" and to sync your fork with the upstream repository."